Example failed to build in vscode with go 1.24

### What is the areas you experience the issue in?

Go-COSE Library

### What is not working as expected?

The CWT example does not build as expected in vscode with go 1.24.

```
Running tool: C:\Program Files\Go\bin\go.exe test -timeout 30s -run ^ExampleCWTMessage$ github.com/veraison/go-cose

# github.com/veraison/go-cose_test
# [github.com/veraison/go-cose_test]
d:\Source\Repos\go-cose\cwt_test.go:13:1: ExampleCWTMessage refers to unknown identifier: CWTMessage
FAIL	github.com/veraison/go-cose [build failed]
FAIL
```

### What did you expect to happen?

```
Running tool: C:\Program Files\Go\bin\go.exe test -timeout 30s -run ^Example_cWTMessage$ github.com/veraison/go-cose

ok  	github.com/veraison/go-cose	3.599s
```

### How can we reproduce it?

Run the `ExampleCWTMessage` in vscode by clicking `run test`.

### Describe your environment

```console
> go version
go version go1.24.0 windows/amd64
```

### What is the version of your Go-COSE Library?

`main` branch with commit `92fbe9502b2c7e2fcc4300348e023088ef3754a7`.

--- a/cwt_test.go
+++ b/cwt_test.go
@@ -10,7 +10,7 @@ import (
 )
 
 // This example demonstrates signing and verifying COSE_Sign1 signatures.
-func Example_cWTMessage() {
+func ExampleCWTClaims() {
 	// create message to be signed
 	msgToSign := cose.NewSign1Message()
 	msgToSign.Payload = []byte("hello world")

--- a/cwt_test.go
+++ b/cwt_test.go
@@ -10,7 +10,7 @@ import (
 )
 
 // This example demonstrates signing and verifying COSE_Sign1 signatures.
-func ExampleCWTMessage() {
+func Example_cWTMessage() {
 	// create message to be signed
 	msgToSign := cose.NewSign1Message()
 	msgToSign.Payload = []byte("hello world")

